pkg/display/display.go

```diff
@@ -278,7 +278,12 @@ func displayCSV(ctx context.Context, result *queryresult.Result) {
 	// print the data as it comes
 	// define function display each csv row
 	rowFunc := func(row []interface{}, result *queryresult.Result) {
-		rowAsString, _ := ColumnValuesAsString(row, result.Cols)
+		columns := result.Cols
+		rowAsString := make([]string, len(columns))
```

Comment:
`rowAsString := ColumnValueAsString(val, col, WithNullString(""))`

where we change `ColumnValueAsString` to accept a variadic set of options.